src/reducers/index.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers } from 'redux';
+import books from './books';
+// rootReducer
+/*
+const rootReducer = combineReducers({
```

Comment:
- [ ] Comments: It’s better to delete commented piece of codes. If you need to review a previous version of your code, use git history 

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png)